src/m1_facilitator/services/ConfirmWithdrawService.ts

```diff
@@ -146,21 +140,21 @@ export default class ConfirmWithdrawService extends Observer<Gateway> {
       [message.messageHash],
     );
 
-    const erc20GatewayTokenPairRecord = await this.erc20GatewayTokenPairRepository.get(
-      gatewayAddress,
-      withdrawIntent.tokenAddress as string,
+    const utilityTokenInteract = Mosaic.interacts.getUtilityToken(
+      this.auxiliaryWeb3,
+      withdrawIntent.tokenAddress,
     );
 
-    const utilityToken = erc20GatewayTokenPairRecord && erc20GatewayTokenPairRecord.utilityToken;
-
+    const valueToken = await utilityTokenInteract.methods.valueToken().call();
+    Logger.debug(`ConfirmWithdrawService:: Storage Proof s${proof.storageProof[0]}`);
```

Comment:
Do we need this log. It will be large object.

Reply:
I help in debugging. It's a debug log, default disabled.